Update docs with new US2 content (#17870)
getsentry/sentry-docs: docs/organization/data-storage-location/index.mdx

**Please note that once selected, your data storage location can’t be changed. The only way to switch it is by creating a new organization.**

> New US organizations may automatically be placed in the US2 region. You can check which region your organization is in by viewing your organization’s settings page.

Reviewer: Is this meant to be intentionally vague? It makes it sound like we're not sure exactly where US organizations are routed. Do we have a list of possible regions US organizations may be placed? 

I may reword to something more like "New US organizations are automatically placed in one of the following regions: [list of regions]. You can check which region your organization is in by viewing your organization’s settings page." - If this is true.

Author: We can tighten up the verbiage a bit, but we wanted to leave the door open as to which exact US region new orgs will be created in since we plan on ramping up from 1% of new org creations -> 100% over a few days/weeks.

Reviewer: @BojanOro , are you saying that we'll be ramping up from 1 - 100% in even more regions? If it's just US and US2, the above seems fine. We can always update when we create more regions, if that happens. 

It's not explicitly saying that they *will* be placed in a particular region, just that they will be placed in *one* of these regions. Is that not accurate?

Author: Upon registration, customers will be able to select US or DE. If they select US, then we will assign them either US or US2. The current plan is assign 1% of new customers to US2, and then gradually increase. That means that customers will be randomly assigned either US or US2. Eventually we'll get to a point where 100% of new customers go to US2.

Reviewer: >Eventually we'll get to a point where 100% of new customers go to US2.

Do we have a set date for that? I would suggest this: 

"New US organizations are automatically placed in either US or US2. You can check which region your organization is in by viewing your organization’s settings page."

Once 100% of new customers will go to US2, we can update to: 

"New US organizations created after [the date we are at 100%] are automatically placed in US2. You can check which region your organization is in by viewing your organization’s settings page."

> New US organizations may automatically be placed in the US2 region. You can check which region your organization is in by viewing your organization’s settings page.

Author: ```suggestion
New US organizations are automatically placed in either US or US2. You can check which region your organization is in by viewing your organization’s settings page.
```

Your selection determines your data storage location only. Sentry will continue to access and process your data in accordance with your agreement with Sentry for the Sentry service and our [Privacy Policy](https://sentry.io/privacy/).

The table below shows the location where your data will be stored at rest, based on your selection.

| **Data Storage Location**     | **Region** | **Physical Location**  |
| ----------------------------- | ---------- | ---------------------- |
| United States of America (US) | US         | Iowa, USA              |
| United States of America (US) | US2        | Las Vegas, Nevada, USA |
| European Union (EU)           | EU         | Frankfurt, Germany     |

## What Types of Data Are Stored Where

> Here’s a list of the types of data that will be stored in whichever data storage location (US or EU) you select.

Reviewer 2: @BojanOro Do we need to explicitly call out US and US2 , as our AI overlord above suggests?

> Here’s a list of the types of data that will be stored in whichever data storage location (US or EU) you select.

Reviewer 3: I don't think so. In the table above US2 is also a 'US' data storage location.
